datastore/db/bytebase/FertiScan/schema_0.0.13.sql

```diff
@@ -0,0 +1,324 @@
+    "unit_id" uuid REFERENCES "fertiscan_0.0.13".unit(id),
+    "metric_type" "fertiscan_0.0.13".metric_type,
+    "label_id" uuid REFERENCES "fertiscan_0.0.13".label_information(id)
+    );
+
+    CREATE TABLE "fertiscan_0.0.13"."sub_type" (
+        "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+        "type_fr" text Unique NOT NULL,
+        "type_en" text unique NOT NULL
+    );
+
+
+    CREATE TABLE "fertiscan_0.0.13"."specification" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "humidity" float,
+    "ph" float,
+    "solubility" float,
+    "edited" boolean,
+    "label_id" uuid REFERENCES "fertiscan_0.0.13".label_information(id),
+    "language" "fertiscan_0.0.13".LANGUAGE
+    );
+
+    CREATE TABLE "fertiscan_0.0.13"."sub_label" (
+        "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+        "text_content_fr" text NOT NULL DEFAULT '',
+        "text_content_en" text NOT NULL DEFAULT '',
+        "label_id" uuid NOT NULL REFERENCES "fertiscan_0.0.13"."label_information" ("id"),
+        "edited" boolean, --this is because with the current upsert we can not determine if it was edited or not
+        "sub_type_id" uuid NOT NULL REFERENCES "fertiscan_0.0.13"."sub_type" ("id")
+    );
+
+    CREATE TABLE "fertiscan_0.0.13"."micronutrient" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "read_name" text,
+    "value" float,
+    "unit" text ,
+    "element_id" int REFERENCES "fertiscan_0.0.13".element_compound(id),
+    "label_id" uuid REFERENCES "fertiscan_0.0.13".label_information(id),
+    "edited" boolean,
+    "language" "fertiscan_0.0.13".LANGUAGE
+    );
+
+    CREATE TABLE "fertiscan_0.0.13"."guaranteed" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "read_name" text,
+    "value" float ,
+    "unit" text ,
+    "language" "fertiscan_0.0.13".LANGUAGE,
+    "element_id" int REFERENCES "fertiscan_0.0.13".element_compound(id),
+    "label_id" uuid REFERENCES "fertiscan_0.0.13".label_information(id),
+    "edited" boolean
+    );
+
+    CREATE TABLE "fertiscan_0.0.13"."ingredient" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "organic" boolean,
+    "active" boolean,
+    "name" text,
+    "value" float,
+    "unit" text,
+    "edited" boolean,
+    "label_id" uuid REFERENCES "fertiscan_0.0.13".label_information(id),
+    "language" "fertiscan_0.0.13".LANGUAGE
+    );
+
+    CREATE TABLE "fertiscan_0.0.13"."inspection" (
+    "id" uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+    "verified" boolean DEFAULT false,
+    "upload_date" timestamp DEFAULT CURRENT_TIMESTAMP,
+    "updated_at" timestamp DEFAULT CURRENT_TIMESTAMP,
+    "inspector_id" uuid NOT NULL REFERENCES "fertiscan_0.0.13".users(id),
+    "label_info_id" uuid REFERENCES "fertiscan_0.0.13".label_information(id),
+    "sample_id" uuid REFERENCES "fertiscan_0.0.13".sample(id),
+    "picture_set_id" uuid REFERENCES "fertiscan_0.0.13".picture_set(id),
+    "user_commment" text,
+    "original_dataset" json
```

Comment:
Not sure how `original_dataset` crept in here again but as long as operational transactions are not dependent on it, this is fine by me.